src/System.Text.JsonLab.Serialization/System/Text/Json/Serialization/JsonConverter.FromJson.Stream.cs

```diff
@@ -15,26 +15,26 @@ public static partial class JsonConverter
     {
         private const int HalfMaxValue = int.MaxValue / 2;
 
-        public static Task<T> FromJsonAsync<T>(this Stream stream, JsonConverterSettings settings = null, CancellationToken cancellationToken = default)
+        public static Task<T> FromJsonAsync<T>(this Stream reader, JsonConverterSettings settings = null, CancellationToken cancellationToken = default)
```

Comment:
Why this change?

Reply:
To align with PipeWriter and PipeReader using `writer` and `reader` naming.

Reply:
I don't think they should align

Reply:
OK I'll bring it up in the API review tomorrow.

Reply:
Actually using `Stream stream` is more consistent with other code, so I'll make that change now.